apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -82,16 +91,112 @@ class ConnectionInstructions extends React.Component {
           <li>{applabI18n.makerSetupWebSerialConnectToComputer()}</li>
           <li>{applabI18n.makerSetupWebSerialConnectToBoardButton()}</li>
           <li>
-            <SafeMarkdown
-              markdown={applabI18n.makerSetupWebSerialWindowConnect()}
-            />
+            {applabI18n.makerSetupWebSerialWindowConnect()}
+            <ul className={styles.troubleshootList}>
+              <li>
+                <SafeMarkdown
+                  markdown={applabI18n.makerSetupNoCompatibleDevices({
+                    installInstructionsId: styles.installInstructionsId,
+                  })}
+                />
+              </li>
+            </ul>
           </li>
           <li>{applabI18n.makerSetupWebSerialSuccessfulConnection()}</li>
         </ol>
         <SafeMarkdown
           markdown={applabI18n.makerSetupWebSerialSupportArticle()}
+          openExternalLinksInNewTab={true}
         />
         {connectionState}
+        <CPExpressInstallInstructions />
+      </div>
+    );
+  }
+}
+
+class CPExpressInstallInstructions extends React.Component {
```

Comment:
What do you think of moving `ConnectionInstructions` and this new component into separate files? Don't have strong opinion about this but `SetupInstructions` is starting to get large.

Reply:
I kinda want to keep them together because they have a bit of shared information between them (the jumplinks between the two components and the styling for the troubleshooting lists). I spose I could refactor that stuff tho so both can use it. Lemme noodle on it.

Reply:
Keeping them together sounds reasonable to me!